tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -103,10 +103,17 @@ def update_encrypted_fields(self, is_encrypted):
                 if type(field) == TransformField and "is_encrypted" in field.kwargs:
                     field.kwargs['is_encrypted'] = is_encrypted
 
-def contains_encrypted_indicator(line, encryption_field):
+def contains_encrypted_indicator(line, encryption_val):
     """Determine if line contains encryption indicator."""
-    if encryption_field is not None:
-        return encryption_field.parse_value(line) == "E"
+    if encryption_val is not None:
+        return encryption_val == "E"
+    return False
+
+def contains_tribe_code(line, tribe_code_val):
+    """Determine if the line has a valid tribal code."""
+    options = {"   ", None, "000"}
+    if tribe_code_val is not None:
```

Comment:
I'm confused on checking for `None` but then it exists in `options`. Can you elaborate?

Reply:
Just me not thinking. The branch is unnecessary!